chart/README.md

```diff
@@ -0,0 +1,74 @@
+| turing.config.router.fiberDebugLog | bool | `true` | Enable debugging for Fiber library used by Turing router |
+| turing.config.router.fluentd.flushIntervalSeconds | int | `90` | How often should Fluentd flush the buffered log |
+| turing.config.router.fluentd.image | string | `nil` | Docker image for Fluentd log forwarder. User is expected to specify the Fluentd image for now as there is no publicly available image.  Currently this is required only if users needs to save Turing logs in BigQuery. |
+| turing.config.router.image.registry | string | `"docker.io/"` | Docker registry for Turing router image. User is expected to override the registry for now as there is no publicly available Turing image |
+| turing.config.router.image.repository | string | `"turing-router"` | Docker image repository for Turing router |
+| turing.config.router.image.tag | string | `"latest"` | Docker image tag for Turing router |
+| turing.config.router.jaeger.collectorEndpoint | string | `nil` | Jaeger tracing collector endpoint  |
+| turing.config.router.jaeger.enabled | bool | `false` | Enable Jaeger tracing |
+| turing.config.router.logLevel | string | `"DEBUG"` | Log level for Turing router |
+| turing.config.sentry.dsn | string | `nil` | Data source name for the Sentry project  |
+| turing.config.sentry.enabled | bool | `false` | Enable integration with Sentry application monitoring https://sentry.io |
+| turing.config.serviceAccount.enabled | bool | `false` | Enable usage of Google Cloud service account JSON key |
+| turing.config.serviceAccount.secretKey | string | `nil` | Secret key for Google Cloud service account JSON key |
+| turing.config.serviceAccount.secretName | string | `nil` | Secret name for Google Cloud service account JSON key |
+| turing.config.vault.address | string | `nil` | Vault server address (required) |
+| turing.config.vault.token | string | `nil` | Vault authentication token (required) |
+| turing.image.registry | string | `"docker.io/"` | Docker registry for Turing API image. User is required to override the registry for now as there is no publicly available Turing image |
+| turing.image.repository | string | `"turing"` | Docker image repository for Turing API |
+| turing.image.tag | string | `"latest"` | Docker image tag for Turing API |
+| turing.livenessProbe.path | string | `"/v1/internal/live"` | HTTP path for liveness check |
+| turing.readinessProbe.path | string | `"/v1/internal/ready"` | HTTP path for readiness check |
+| turing.resources | object | `{}` | Resources requests and limits for Turing API. This should be set  according to your cluster capacity and service level objectives. Reference: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ |
+| turing.service.externalPort | int | `8080` | Turing API Kubernetes service port number |
+| turing.service.internalPort | int | `8080` | Turing API container port number |
```

Comment:
It may be a good idea to have a separate section for the required config options. Something like `Configuration` (required values go here) and `Advanced Config Options` (the rest of values go here). 

This will make it easier for users to prepare minimal required values file for their deployments.

Reply:
Yeah we can have another section in the documentation showing example deployments with minimal and advanced configuration. We can add this in another pull request.

I'd actually prefer not to have any required configuration by **default**. Any required configuration should require the user to **specifically enable** specific chart component. For instance, token key for NewRelic is required only if the user specifically enable NewRelic integration. 

But I think we need some refactoring in our config format and loading first.